erpnext/accounts/doctype/purchase_invoice_item/purchase_invoice_item.json

```diff
@@ -67,6 +67,12 @@
   "total_weight",
   "column_break_38",
   "weight_uom",
+  "cultivation_tax_details_section",
```

Comment:
```suggestion
  "cultivation_tax_details",
```

Reply:
Hi Sahil,

Updated the PR as per suggestions.

PR link: https://github.com/Bloomstack/erpnext/pull/1603